### PR TITLE
refactor: code of conduct

### PR DESCRIPTION
**Pull request by basiima**

### CODE REVIEW

File: components/HeroSection.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ export default function HeroSection() {
         className="w-full h-screen bg-no-repeat bg-cover "
         style={{
           backgroundImage:
-            "url(https://user-images.githubusercontent.com/65954740/227708763-5d987ff4-564e-42e0-a8f8-2d609611fccb.jpg)",
+            "url(https://i.ibb.co/v1cVrYd/Untitled.jpg)",
```
Comment (HassanBahati) [inline components/HeroSection.js:8]: @basiima was this image able to display?

Comment (HassanBahati) [reply components/HeroSection.js:8]: Images from external sources dont work unless registered in `next.config.js`

Comment (basiima) [reply components/HeroSection.js:8]: it worked

**Comment by HassanBahati** [comment]
Oh yah I've  seen the preview

**Review by HassanBahati — APPROVED** [review]
Looks good

**Pull request merged**